content/applications/finance/accounting/receivables/customer_invoices/electronic_invoicing.rst

```diff
@@ -0,0 +1,33 @@
+====================
+Electronic invoicing
+====================
+
+EDI, or electronic data interchange, allows to rapidly send information digitally using standard
+formats.
+
+This feature enables automating the administration between companies and might also be required by
+some governments for fiscal control or to facilitate the administration.
+
+Any business that processes large volumes of documents and forms can benefit from an EDI solution to
+increase automation and improve accuracy.
+
+.. seealso::
+   - `Odoo and EDI <https://www.odoo.com/blog/odoo-news-5/odoo-and-edi-766#blog_content>`_
+   - :ref:`Fiscal localization packages <overview/fiscal-localization-package>`
+   - :ref:`List of supported countries <overview/localizations-list>`
+
+Configuration
+=============
+
+Go to :menuselection:`Accounting --> Configuration --> Journals --> Customer Invoices -->
+Advanced Settings --> Electronic Invoicing` and check the boxes you need.
```

Comment:
instead of "check boxes" maybe "select the formats you want to enable for this journal" or something like that?
+ small screenshot can't hurt here.